src/main/resources/avro/referencemethods.avdl

```diff
@@ -284,6 +355,12 @@ record GetSequenceBasesRequest {
   union { null, long } end = null;
 
   /**
+  Specifies the maximum number of bases to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
```

Comment:
That's probably true. I let @macieksmuga talk me into trying to fix what we thought was an oversight, but if we're going to be discussing it at all it needs to be its own issue.